test/itest_testreorg.py

```diff
@@ -0,0 +1,30 @@
+import testlib
+
+# So the test roughly goes as follows:
+#  Start lit up normally, generate 10 blocks using bitcoind's generate command
+# Store the first hash and invalidate that, creating a 9 block re-org. Then
+# Generate block by block and see if lit reorgs. Lit should reorg only after
+# we produce 9 blocks
+def run_test(env):
+    print("REUNNINSA")
+    bc = env.bitcoind
+    lit1 = env.lits[0]
+    bc.rpc.generate(11)
+    tip = bc.rpc.getblockcount()
+    print("TIP", tip)
+    targetblockhash = bc.rpc.getblockhash(tip - 9)
+    # TODO: Stop and start lit again here
```

Comment:
This isn't `# TODO` anymore is it?

Reply:
yeah, after #420 is merged, will do a rebase and then fix this

Reply:
Also I think this file can just be named "reorg" instead of "testreorg" since that's a little redundant.

Reply:
Oh yeah, that makes sense. Will rename and put in the restart code as well.